lib/iris/experimental/um.py

```diff
@@ -713,3 +715,130 @@ def close(self):
                     os.rename(tmp_file.name, self.filename)
             finally:
                 self._source.close()
+
+
+def cutout(ffv_src, filename, cutout):
+    """
+    Subset a FieldsFileVariant to a specified i, j index and number of points
+    along each axis.
+
+    Args:
+
+    * ffv_src (:class:`FieldsFileVariant`)
+        A source FieldFileVariant to subset.
+    * filename (string)
+        A name of a file to save the subset FieldsFileVariant to.
+    * cutout (list)
+        The cutout parameters [x index, y index, number of x points, number
+        of y points].)
+
+    Returns:
+        An instance of :class:`FieldsFileVariant` in `CREATE_MODE`.
+
+    """
+
+    dx = ffv_src.real_constants[0]
+    dy = ffv_src.real_constants[1]
+    zx0 = ffv_src.real_constants[2]
+    zy0 = ffv_src.real_constants[3]
+    nx0 = ffv_src.integer_constants[5]
+    ny0 = ffv_src.integer_constants[6]
+
+    zx1, zy1, nx1, ny1 = _values_after_cutout(dx, dy, zx0, zy0, nx0, ny0,
+                                              cutout)
+
+    ffv_dest = FieldsFileVariant(filename, FieldsFileVariant.CREATE_MODE)
+
+    ffv_dest.fixed_lendth_header = ffv_src.fixed_length_header
```

Comment:
Typo! This should say:
ffv_dest.fixed_**length**_header = ffv_src.fixed_length_header

With that change I believe it should work.